### DISCUSSION TITLE
Is ROS2 supported?

### DISCUSSION THREAD

**Discussion by Coderx7** [microsoft/AirSim#3830] [Support Q&A]
IS ROS2 supported in airsim? 
I have noticed PX4 has switched to ROS2 in their latest version on noetic (ubuntu 20.04) so I just wanted to know if all this is supported. 
Thanks a lot in advance

**Comment by alonfaraj** [comment]
@Coderx7 maybe this PR (https://github.com/microsoft/AirSim/pull/3976) might help you for now
